src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -263,14 +266,17 @@ class FunSpec private constructor(builder: Builder) {
 
     fun receiver(receiverType: KClass<*>) = receiver(receiverType.asTypeName())
 
-    fun returns(returnType: TypeName) = apply {
+    @JvmOverloads fun returns(returnType: TypeName, kdoc: CodeBlock? = null) = apply {
       check(!name.isConstructor && !name.isAccessor) { "$name cannot have a return type" }
       this.returnType = returnType
+      kdoc?.let {
+        this.returnKdoc.add("@return %L", it)
+      }
     }
 
-    fun returns(returnType: Type) = returns(returnType.asTypeName())
+    @JvmOverloads fun returns(returnType: Type, kdoc: CodeBlock? = null) = returns(returnType.asTypeName(), kdoc)
 
-    fun returns(returnType: KClass<*>) = returns(returnType.asTypeName())
+    @JvmOverloads fun returns(returnType: KClass<*>, kdoc: CodeBlock? = null) = returns(returnType.asTypeName(), kdoc)
```

Comment:
How about providing overloads that accept `kdoc` as a `String`? This way you won't need to create a `CodeBlock` for simple cases.

Reply:
Done!